dashboard/compliance/mitre_attack_azure.py

```diff
@@ -0,0 +1,23 @@
+import warnings
+
+from dashboard.common_methods import get_section_containers_format2
+
+warnings.filterwarnings("ignore")
+
+
+def get_table(data):
+    aux = data[
+        [
+            "REQUIREMENTS_ID",
+            "REQUIREMENTS_SUBTECHNIQUES",
+            "CHECKID",
+            "STATUS",
+            "REGION",
+            "ACCOUNTID",
+            "RESOURCEID",
+        ]
+    ].copy()
```

Comment:
What is the purpose of the copy here?

Reply:
Same as above

Reply:
This is not a valid answer 😂 If we are copying all the objects for nothing is something that needs to be reviewed.